### PR TITLE
Delete S3693 (deprecated)

### PR DESCRIPTION
**Pull request by costin-zaharia-sonarsource**

### CODE REVIEW

File: analyzers/tests/SonarAnalyzer.UnitTest/PackagingTests/CsRuleTypeMapping.cs [hunk 1]
```diff
@@ -3690,7 +3690,7 @@ internal static class CsRuleTypeMapping
             //["3690"],
```
Comment (csaba-sagi-sonarsource) [inline analyzers/tests/SonarAnalyzer.UnitTest/PackagingTests/CsRuleTypeMapping.cs:3690]: This same change in VbRuleTypeMapping is missing.

**Review by csaba-sagi-sonarsource — CHANGES_REQUESTED** [review]
One more change needed.

File: analyzers/tests/SonarAnalyzer.UnitTest/PackagingTests/CsRuleTypeMapping.cs (hunk 1 above)
Comment (costin-zaharia-sonarsource) [reply analyzers/tests/SonarAnalyzer.UnitTest/PackagingTests/CsRuleTypeMapping.cs:3690]: Nice catch! Thanks!

**Review by csaba-sagi-sonarsource — APPROVED** [review]
LGTM!

**Pull request merged**